enhancements/two-node-fencing/tnf.md

```diff
@@ -551,6 +551,20 @@ This section provides specific steps for how two-node clusters would handle inte
    3. Stop failure is optionally escalated to a node failure (fencing)
    4. Start failure defaults to leaving the service offline
 
+#### Running Two Node OpenShift with Fencing with a Failed Node
+
+An interesting aspect of TNF is that should a node fail and remain in a failed state, the cluster recovery operation will allow the survivor to restart etcd as a cluster-of-one and resume normal
+operations. In this state, we have an operation running cluster with a single control plane node. So, what is the difference between this state and running Single Node OpenShift? There are three key aspects:
+
+1. Operators that deploy to multiple nodes will become degraded.
+2. Operations that would violate pod-disruption budgets will not work.
+3. Lifecycle operations that would violate the `MaxUnavailable` setting of the control-plane MachineSet cannot proceed. This includes MCO node reboots and cluster upgrades.
```

Comment:
Do we explain how will this be done  ?

Reply:
This happens automatically because the logic related to the Machine config's MaxUnavailable default is set to 1. Basically MCO will not signal for the machine to reboot if it would violate this. I don't know that there is much to explain. I could link it to the appropriate documentation - that might be helpful.